scalanativelib/api/src/ScalaNativeWorkerApi.scala

```diff
@@ -52,6 +52,8 @@ sealed abstract class ReleaseMode(val name: String)
 object ReleaseMode {
   case object Debug extends ReleaseMode("debug")
   case object Release extends ReleaseMode("release")
+  case object ReleaseFast extends ReleaseMode("release-fast")
+  case object ReleaseFull extends ReleaseMode("release-full")
```

Comment:
These two modes don't work with SN 0.3. Since when I updated mill I joined the SN 0.4 and 0.3 workers I think it's fine. When 0.4.0 is out we can remove the `Release` mode at all.

Reply:
Would be great to have these information as comment in the code, so anybody can see it when deciding which mode to choose.